File: Advanced Business Development With dotNET/Models/Patio.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MottuFlowApi.Models
{
    [Table("patio")]
    public class Patio
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("id_patio")]
        public int IdPatio { get; set; }

        [Required]
        [MaxLength(100)]
        [Column("nome")]
        public string Nome { get; set; } = string.Empty;

        [Required]
        [MaxLength(200)]
        [Column("endereco")]
        public string Endereco { get; set; } = string.Empty;

        [Required]
        [Column("capacidade_maxima")]
        public int CapacidadeMaxima { get; set; }

        // Navigation
        public List<Moto> Motos { get; set; } = new List<Moto>();
        public List<Camera> Cameras { get; set; } = new List<Camera>();
        public List<Localidade> Localidades { get; set; } = new List<Localidade>();
    }
}

File: DevOps Tools and Cloud Computing/MottuFlow/src/main/java/com/sprint/MottuFlow/domain/patio/Patio.java
package com.sprint.MottuFlow.domain.patio;

import com.sprint.MottuFlow.domain.camera.Camera;
import com.sprint.MottuFlow.domain.localidade.Localidade;
import com.sprint.MottuFlow.domain.moto.Moto;
import jakarta.persistence.*;
import lombok.*;

import java.util.List;

@Entity
@Table(name = "patio")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Patio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idPatio;

    @Column(nullable = false, length = 100)
    private String nome;

    @Column(nullable = false, length = 200)
    private String endereco;

    @Column(nullable = false)
    private int capacidadeMaxima;

    @OneToMany(mappedBy = "patio", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Moto> motos;

    @OneToMany(mappedBy = "patio", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Camera> cameras;

    @OneToMany(mappedBy = "patio", cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Localidade> localidades;


	public Long getIdPatio() {
		return idPatio;
	}

	public void setIdPatio(Long idPatio) {
		this.idPatio = idPatio;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getEndereco() {
		return endereco;
	}

	public void setEndereco(String endereco) {
		this.endereco = endereco;
	}

	public int getCapacidadeMaxima() {
		return capacidadeMaxima;
	}

	public void setCapacidadeMaxima(int capacidadeMaxima) {
		this.capacidadeMaxima = capacidadeMaxima;
	}

	public List<Moto> getMotos() {
		return motos;
	}

	public void setMotos(List<Moto> motos) {
		this.motos = motos;
	}

	public List<Camera> getCameras() {
		return cameras;
	}

	public void setCameras(List<Camera> cameras) {
		this.cameras = cameras;
	}

	public List<Localidade> getLocalidades() {
		return localidades;
	}

	public void setLocalidades(List<Localidade> localidades) {
		this.localidades = localidades;
	}
}
